CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,3 @@
 - [cygnus-common][feature] Add /v1/namemappings route to API (#1277)
 - [cygnus-common][bugfix] Support empty servicenames in mamemappings 
-
+- [cygnus-common][docker] Truncate to 0.5GB sth collections
```

Comment:
Maybe better?

[cygnus-ngsi][docker] Set default truncate size to 0.5GB for sth collections

(I guess cygnus-ngsi as the agent.conf file is inside cygnus-ngsi/ directory)

Reply:
fixed in 3bdd6c58130d45d4a37e8f42c7702db35064f602